### PR TITLE
CHAOS-5501 Linux JVM trigger gc

### PR DESCRIPTION
**Pull request by SmritiSatya**
Linux JVM trigger gc docs and fault diagram

### CODE REVIEW

File: docs/chaos-engineering/chaos-faults/linux/linux-jvm-trigger-gc.md [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+    <th> Notes </th>
+  </tr>
+  <tr>
+    <th> Pid </th>
+    <td> The process Id that Byteman uses to target the service. This is mutually exclusive with <b>Startup command</b>. </td>
+    <td> For example, <code>6429</code>. For more information, go to <a href="#pid"> process Ids</a>.</td>
+  </tr>
+  <tr>
+    <th> Startup command </th>
+    <td> The path to the JAR file or the Java application. This is mutually exclusive with <b>Pid</b>.</td>
```
Comment (neelanjan00) [inline docs/chaos-engineering/chaos-faults/linux/linux-jvm-trigger-gc.md:40]:
```suggestion
    <td> The command used to start the Java process. A substring match is performed for the given value for all the processes. This is mutually exclusive with <b>Pid</b>.</td>
```

File: docs/chaos-engineering/chaos-faults/linux/linux-jvm-trigger-gc.md [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+
+### Mandatory tunables
+<table>
+  <tr>
+    <th> Tunable </th>
+    <th> Description </th>
+    <th> Notes </th>
+  </tr>
+  <tr>
+    <th> Pid </th>
```
Comment (neelanjan00) [inline docs/chaos-engineering/chaos-faults/linux/linux-jvm-trigger-gc.md:34]: Please check in all the other places too, `<th>` should only be used in heading cells.
```suggestion
    <td> Pid </td>
```

File: docs/chaos-engineering/chaos-faults/linux/linux-jvm-trigger-gc.md [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+    <th> Notes </th>
+  </tr>
+  <tr>
+    <th> Duration </th>
+    <td> Duration through which chaos is injected into the target resource. Should be provided in <code>[numeric-hours]h[numeric-minutes]m[numeric-seconds]s</code> format. </td>
+    <td> Default: <code>30s</code>. Examples: <code>1m25s</code>, <code>1h3m2s</code>, <code>1h3s</code>. For more information, go to <a href="/docs/chaos-engineering/chaos-faults/common-tunables-for-all-faults/#duration-of-the-chaos"> duration of the chaos.</a></td>
+  </tr>
+  <tr>
+    <th> Port </th>
+    <td> Port used to Byteman to start the Byteman agent. </td>
```
Comment (neelanjan00) [inline docs/chaos-engineering/chaos-faults/linux/linux-jvm-trigger-gc.md:59]:
```suggestion
    <td> Port to be used by the Byteman agent. </td>
```

File: docs/chaos-engineering/chaos-faults/linux/linux-jvm-trigger-gc.md [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+    <td> For example, <code>9091</code>. </td>
```
Comment (neelanjan00) [inline docs/chaos-engineering/chaos-faults/linux/linux-jvm-trigger-gc.md:60]:
```suggestion
    <td> Default: <code>9091</code>. </td>
```

File: docs/chaos-engineering/chaos-faults/linux/linux-jvm-trigger-gc.md [hunk 5]
```diff
@@ -0,0 +1,122 @@
...
+  </tr>
+  <tr>
+    <th> Ramp time </th>
```
Comment (neelanjan00) [inline docs/chaos-engineering/chaos-faults/linux/linux-jvm-trigger-gc.md:63]: Please write all of these in camelCase, that's how the input is provided in the manifest too.

Check in other places too.
```suggestion
    <th> rampTime </th>
```

File: docs/chaos-engineering/chaos-faults/linux/linux-jvm-trigger-gc.md [hunk 6]
```diff
@@ -0,0 +1,122 @@
...
+    <td> Period to wait before and after injecting chaos. Should be provided in <code>[numeric-hours]h[numeric-minutes]m[numeric-seconds]s</code> format. </td>
+    <td> Default: <code>0s</code>. Examples: <code>1m25s</code>, <code>1h3m2s</code>, <code>1h3s</code>. For more information, go to <a href= "/docs/chaos-engineering/chaos-faults/common-tunables-for-all-faults#ramp-time">ramp time.</a></td>
+  </tr>
+  <tr>
+    <td>  </td>
+    <td>  </td>
+    <td>  </td>
+  </tr>
```
Comment (neelanjan00) [inline docs/chaos-engineering/chaos-faults/linux/linux-jvm-trigger-gc.md:71]: We can remove this?

Comment (neelanjan00) [reply docs/chaos-engineering/chaos-faults/linux/linux-jvm-trigger-gc.md:71]: Please check in other faults too.

File: docs/chaos-engineering/chaos-faults/linux/linux-jvm-trigger-gc.md (hunk 2 above)
Comment (SmritiSatya) [reply docs/chaos-engineering/chaos-faults/linux/linux-jvm-trigger-gc.md:34]: The changes to html elements is intentional, the tunables are highlighted this way and easy to identify them.

**Pull request merged**